scripts/snap/impl/snapToJbMain.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -e
+
+if [ -z "$1" ]; then
+echo "Specify the snapping subfolder. For example: ./snapToJbMain.sh androidx/compose-ui/1.6.0-alpha02 compose ':(exclude)compose/material3'"
```

Comment:
Description for args is not clear. What is branch? Out? In? Is it possible to specify hash? What is folder? What should be current git branch?

Reply:
This shouldn't be used explicitly. Added a comment to look the parent scripts/readme